code/modules/projectiles/guns/projectile/constructable/siegecannon.dm

```diff
@@ -34,6 +35,8 @@
 		return 1
 	if(W.is_wrench())
 		wrenchAnchor(user, W, 5)	//Half a second to wrench. Being able to turn it via verb while anchored is intentional.
+	else if((istype(W, /obj/item/weapon/stamp/clown) || istype(W, /obj/item/toy/crayon/rainbow)) && !beenClowned)
```

Comment:
seems weird to use a stamp since consistency with clownification of guns would be just the crayon

Reply:
Mostly I wanted a bit more stamp relevance. I can change it if you'd like but I didn't see the harm in letting it be either.

Reply:
The clownstamp is banana yellow, so you'd need to reflect that in the sprite when making using the stamp to dye it, or my immulshons would be destroyed.

Reply:
I'd like to keep it as both I think, just because it will already be a niche item. A clownified variant of a variant of a makeshift weapon, layers and layers of obscurity there.